docker.local/bin/build.miners.apple.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+GIT_COMMIT=$(git rev-list -1 HEAD)
+echo $GIT_COMMIT
+
+ROOT="$(git rev-parse --show-toplevel)"
+DOCKERDIR="$ROOT/docker.local/build.miner"
+DOCKERFILE="$DOCKERDIR/Dockerfile"
+DOCKERCOMPOSE="$DOCKERDIR/docker-compose.yml"
+
+if [[ "$*" == *"--dev"* ]]
+then
+    echo -e "\nDevelopment mode: building miner locally\n"
+
+    cd "$ROOT/code/go/0chain.net/miner/miner"
+    go build -v -tags "bn256 development" \
+        -ldflags "-X 0chain.net/core/build.BuildTag=$GIT_COMMIT"
+
+    sed 's,%COPY%,COPY ./code,g' "$DOCKERFILE.template" > "$DOCKERFILE"
+
+    cd "$ROOT"
+    docker buildx build --platform linux/amd64 --build-arg GIT_COMMIT=$GIT_COMMIT \
```

Comment:
is `linux/amd64` the only different with the other build.miners.sh? Since this is a shell script, they have easy support for flags. So we should really make that a flag. (Dockerfiles do NOT have good support for flags)

Reply:
same for other scripts

Reply:
Fixed